content/en/seachat/seachat-manual/06-workspace/01-workspace-management.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Workspace Management"
+description: "SeaChat Workspace Management"
+lead: ""
+date: 2020-10-06T08:48:45+00:00
+lastmod: 2024-06-27T08:48:45+00:00
+weight: 61
+draft: false
+images: []
+aliases:
+  - /en/seachat/seachat-manual/workspace/01-workspace-management
+  - /seachat/seachat-manual/workspace/01-workspace-management
+---
+
+## Agents
```

Comment:
first show where this is about:

![image](https://github.com/seasalt-ai/wiki/assets/543547/f10b9fdd-b754-44dc-b1be-5161ff9d81f3)